### DISCUSSION TITLE
Key error for rumpowered depo

### DISCUSSION THREAD

**Discussion by DosuWorld** [jc141x/jc141-bash#28] [Q&A]
there is an error with the key for the rumpowered depo on arch 

 this command : sudo pacman-key --recv-keys cc7a2968b28a04b3
 may showing this : ERROR: Remote key not fetched correctly from keyserver.

if anyone else got this error just edit  etc/pacman.conf (sudo gedit) like this : 

```
 disable SigLevel = Never
#LocalFileSigLevel = Optional
#RemoteFileSigLevel = Required
```
You need to comment out any repository-specific SigLevel settings because they override the global settings. This will result in no signature checking

**Comment by hackernoodle** [comment]
did you encounter this on regular archlinux or a derivative??
